src/scss/_compact.scss

```diff
@@ -1,15 +1,10 @@
-////
-/// @group o-table
-/// @link http://registry.origami.ft.com/components/o-table
-////
-
 /// Make a table 'compact'
 @mixin oTableCompact {
-
-	th,
-	td {
-		@include oTypographySize(0);
-		padding: 4px 8px;
+	.o-table--compact {
+		th,
+		td {
+			@include oTypographySize(0);
```

Comment:
same as other type comments ☝️

Reply:
I've updated this to:

```
		th {
			@include oTypographySansBold(0);
		}

		td {
			@include oTypographySans(0);
		}
```

So that progressively loaded fonts are handled correctly.